lib/components/ErrorComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import { tw, tx } from '../twind'
+import type { Languages } from '../hooks/useLanguage'
+import type { PropsWithLanguage } from '../hooks/useTranslation'
+import { useTranslation } from '../hooks/useTranslation'
+import { AlertOctagon } from 'lucide-react'
+
+type ErrorComponentTranslation = {
+  errorOccurred: string
+}
+
+const defaultErrorComponentTranslation: Record<Languages, ErrorComponentTranslation> = {
+  en: {
+    errorOccurred: 'An error occurred'
+  },
+  de: {
+    errorOccurred: 'Ein Fehler ist aufgetreten'
+  }
+}
+
+export type ErrorComponentProps = {
+  errorText?: string,
+  classname?: string
+}
+
+/**
+ * The Component to show when an error occurred
+ */
+export const ErrorComponent = ({
+  language,
+  errorText,
+  classname
+}: PropsWithLanguage<ErrorComponentTranslation, ErrorComponentProps>) => {
+  const translation = useTranslation(language, defaultErrorComponentTranslation)
+  return (
+    <div className={tx('flex flex-col items-center justify-center gap-y-4 w-full h-24', classname)}>
+      <AlertOctagon size={64} className={tw('text-hw-warning-400')}/>
```

Comment:
This would create an association of our icon and an error, which doesn't seem like something we should try intentionally

Reply:
It would be consistent. And why not? It switches from loading to error but the icon stays that way and catches the view.

Remember catching an error is exceptional and actually a positive thing.

Reply:
> It would be consistent. And why not? It switches from loading to error but the icon stays that way and catches the view.
> 
> Remember catching an error is exceptional and actually a positive thing.

On a setup with bad lighting, it would be hard to differentiate the two. And as we are not currently catching errors and just displaying them, it would be a negative context not a positive one.

Reply:
This discussion misses the point. It's still just a suggestion and we obviously have a different understanding of catching erros 😄

Reply:
I don't think so that we have a different understanding of catching errors, but the point is this should only be shown, when errors couldn't be caught, all error resolutions should be done before showing an error.